integr8ly/cloud-resource-operator
# allow existing blobstorage secret to exist for openshift

ensure the openshift provider does not override an existing result
secret with default values.

verification:
- create a secret example-blobstorage-sec with a bucketName key
  set to test and an additional key set to test
- run make cluster/seed/workshop/blobstorage
- ensure the blobstorage custom resource created reconciles
- ensure the custom values set in the secret are kept and all
  other default keys are set to a default value
